.github/workflows/release.yaml

```diff
@@ -299,8 +299,9 @@ jobs:
           branch: ${{ github.ref }}
           workflow: build.yaml
           workflow_conclusion: completed
-          check_artifacts: true
           name: npm-package
+          check_artifacts: false
+          if_no_artifact_found: fail
```

Comment:
These are the defaults but I opted to be explicit for these reasons:
- won't break if defaults change
- maintainers don't have to go to docs to see defaults because we're explicit here